packages/core/src/saml-applications/libraries/saml-applications.ts

```diff
@@ -33,11 +34,15 @@ export const createSamlApplicationsLibrary = (queries: Queries) => {
     },
   } = queries;
 
-  const createSamlApplicationSecret = async (
-    applicationId: string,
-    // Set certificate life span to 3 years by default.
-    lifeSpanInDays = 365 * 3
-  ): Promise<SamlApplicationSecret> => {
+  const createSamlApplicationSecret = async ({
```

Comment:
add some comments for the input parameters. Especially for the `active`, there is a need to notify all the current active secrets that will be deactivated.